### PR TITLE
fix(DB/Creature): Julianne speach

### PR DESCRIPTION
**Pull request by aleigood**
Fix Julianne's first death sound

part of dbc file:
```
9197,10,"A_JulianneAttack","RajJulianneAttack01.wav","RajJulianneAttack02.wav","RajJulianneAttack03.wav","","","","","","","",1,1,1,1,1,1,1,1,1,1,"Sound\Creature\Julianne",0.689999997616,0x0,30.0,60.0,0x0,0,
9198,10,"A_JulianneDeath01","RajJulianneDies01.wav","","","","","","","","","",1,1,1,1,1,1,1,1,1,1,"Sound\Creature\Julianne",0.899999976158,0x0,30.0,60.0,0x0,0,
9199,10,"A_JulianneEnters","RajJulianneEnters01.wav","","","","","","","","","",1,1,1,1,1,1,1,1,1,1,"Sound\Creature\Julianne",0.899999976158,0x0,30.0,60.0,0x0,0,
9200,10,"A_JulianneRes","RajJulianneRes01.wav","","","","","","","","","",1,1,1,1,1,1,1,1,1,1,"Sound\Creature\Julianne",0.899999976158,0x0,30.0,60.0,0x0,0,
```

### CODE REVIEW

**Comment by Kitzunu** [comment]
Please fix travis issue....
https://travis-ci.org/github/azerothcore/azerothcore-wotlk/jobs/670425760#L624

**Comment by Kitzunu** [comment]
Now all builds are passing :) Good job @BarbzYHOOL

File: data/sql/updates/pending_db_world/rev_1585996243671367200.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1585996243671367200');
+
+-- fix Julianne's first death sound
+UPDATE `creature_text` SET `Sound`=9198 WHERE `CreatureID`=17534 and `GroupID`=2 and `ID`=0;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1585996243671367200.sql:4]: when you write SQL commands use uppercase also for "AND"

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1585996243671367200.sql:4]: @aleigood i fixed the "and" for you, next time no :D

**Comment by BarbzYHOOL** [comment]
@Kitzunu you tested?

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**